parmed/amber/_amberparm.py

```diff
@@ -2006,7 +2006,7 @@ def _get_atom_collection_for_alternate_labels(self):
 
     def _label_alternates(self):
         atom_collection = self._get_atom_collection_for_alternate_labels()
-        possible_labels = list('ABCDEF')
+        possible_labels = list('ABCDEFGHIJKLMNOPQRSTUVWXYZ')
```

Comment:
oh, I think yes. Those string matches to the one in python website

https://docs.python.org/3.1/library/string.html